src/training/scheduler.h

```diff
@@ -192,13 +189,12 @@ class Scheduler : public TrainingObserver {
 
   void started() { LOG(info, "Training started"); }
   void finished() {
-    if (getSigtermFlag())
-      LOG(info, "Training interrupted (SIGTERM).");
+    if (gracefulExitRequested())
```

Comment:
I don't see the name change. `earlyExitRequested`?

Reply:
Changed to saveAndExit().